components/productions/create/event.vue

```diff
@@ -58,24 +58,41 @@
       locale() {
         return this.$i18n.locale;
       },
+      truncatedDescription() {
```

Comment:
you could use lodash's [truncate](https://lodash.com/docs/4.17.15#truncate) function, since we already have the dependency to lodash